### DISCUSSION TITLE
Proposal: Allow <see> tag to display custom string.

### DISCUSSION THREAD

**Discussion by artus** [dotnet/csharplang#8757] [Language Ideas]
When documenting your code, allow for the <see> tag to accept an alternative value that should be displayed when looking at the interpreted xml-documentation.

```c#
/// <summary>
/// This is <see cref="String">text</see>.
/// </summary>
```

Should then display:

> This is `text`. 

Instead of

> This is `String`.

This behavior was supported in the past but not anymore, as seen [here](https://softwareengineering.stackexchange.com/questions/48617/how-does-one-pluralize-see-cref-notations).

**Comment by sharwell** [comment]
Related to #1764. Both should probably be done at the same time.
